configurations/config.txt

```diff
@@ -0,0 +1,20 @@
+# configuration file, all config files should be in ./configurations/ folder
+# line starting with '#' characters are ignored, i.e. comments
+# lines cannot be longer than 250 characters, else the world breaks
+
+
+# lines starting with '>' start a new parsing section for a new module
+# e.g. "> Navigation" line indicates all following lines will be parse by
```

Comment:
Where are the module names defined? Maybe add that to the comment.

Reply:
I agree, good point, will update the comments :)

This, however, illustrates an interesting issue with PR reviews. This code is not really new, i.e. is not a contribution of this PR, as the content of this file was in the repo since the beginning. The PR only moved the file into `configurations` folder. Thus, is it fair to request changes to code that other people wrote before this PR branched off of develop?

Reply:
Good point. But I think in general, when you touch old code (even if you just move it), you should check it and fix any simple problems like this. If it was something more complex then it would be okay to leave any refactoring for a separate PR (and potentially the original author of the code).

Reply:
Also, in this case you extended the comment as well, not just copied it. Of course, if you move a lot of stuff around, it might sometimes not be feasible to review and refactor it all but in this case you made edits right next to the part I commented on. So you probably would have reviewed the existing comment anyway to make sure that what you added actually fits.